packages/@okta/vuepress-site/guides/sign-into-mobile-app/configure-packages/android/installsdk.md

```diff
@@ -1,10 +1,12 @@
-Use the library [Okta OIDC](https://bintray.com/okta/com.okta.oidc.android/okta-oidc-android), available through [JCenter](https://bintray.com/bintray/jcenter). To install it, add the following to your `build.gradle`:
+Use the library [Okta OIDC](https://bintray.com/okta/com.okta.oidc.android/okta-oidc-android), available through [JCenter](https://bintray.com/bintray/jcenter).
```

Comment:
Use the library [Okta OIDC](https://bintray.com/okta/com.okta.oidc.android/okta-oidc-android) that is available through [JCenter](https://bintray.com/bintray/jcenter).